### PR TITLE
Using new graylog Vagrantfile as per graylog instructions.

### PR DESCRIPTION
**Pull request by DRvanR**
see: http://docs.graylog.org/en/2.0/pages/installation/vagrant.html

We might want to tweak the memory requirements if needed, for now I'd recommend to keep it as is.

Future step could be to combine the two vagrant files into a single file.

### CODE REVIEW

**Comment by DRvanR** [comment]
@arothuis ping

File: docs/development.md [hunk 1]
```diff
@@ -9,8 +9,12 @@ and follow the installation instructions for each application.
 Useful Postman collections can be found on LastPass under `Shared-StepUp`.
 
 ## Graylog
-In order for Graylog2 to work, the correct source input should be configured. In the `System/Inputs` screen,
-add a GELF UDP with the configured settings, `port: 12201` and `bind_address: 0.0.0.0` by default.
+Graylog should work out of the box. You can log in using the username `admin` and the password `password`.  
+Should no log messages show up, the correct source input should be configured. In the `System/Inputs` screen.
```
Comment (arothuis) [inline docs/development.md]: 👍 for updating the docs!

> In the `System/Inputs` screen.

This sentence can either be removed or re-added to the sentence it was part of.

File: graylog/Vagrantfile [hunk 2]
```diff
@@ -4,46 +4,27 @@
 # Vagrantfile API/syntax version. Don't touch unless you know what you're doing!
 VAGRANTFILE_API_VERSION = "2"
 
-Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
-  config.vm.box = "puphpet/debian75-x64"
-  config.vm.box_version = "2.0"
+Vagrant.configure(2)  do |config|
```
Comment (arothuis) [inline graylog/Vagrantfile]: IMHO, the hardcoded Vagrantfile version number (`Vagrant.configure(2)`) could be extracted as previously was the case.

File: graylog/Vagrantfile [hunk 3]
```diff
@@ -4,46 +4,27 @@
...
+  config.vm.box = "ubuntu/trusty64"
+  config.vm.hostname = "StepUp-log-Server"
```
Comment (arothuis) [inline graylog/Vagrantfile]: 👍 for changing it to the current project name and removing anachronisms, although I'm not entirely sure about the casing of `StepUp`. 
In repository names and namespaces, `Stepup` is used.

Maybe: `Stepup-log-server` or `Stepup Log Server`?

Shouldn't other references to SuaaS in this repository be changed as well -- for the sake of consistency and ubiquitousness? 
- Vagrantfile for the applications
- `suaas.yml`
- `suaas` directory (incl. reference to it in `.gitignore`)
- inventories (incl. reference to it in `graylog.yml` and `suaas.yml`)
- documentation

File: graylog/Vagrantfile (hunk 2 above)
Comment (DRvanR) [inline graylog/Vagrantfile]: disagree, doesn't add any relevance. This is documented behavior and even is generated like this, see also [here][1].

[1]: https://www.vagrantup.com/docs/vagrantfile/version.html

File: docs/development.md (hunk 1 above)
Comment (DRvanR) [inline docs/development.md]: hehe, yeh, `s/./,/` :wink:

File: graylog/Vagrantfile (hunk 3 above)
Comment (DRvanR) [inline graylog/Vagrantfile]: `Stepup-log-server` it'll be, the rest is out of scope for now. Will add tech debt issue.

File: graylog/Vagrantfile (hunk 2 above)
Comment (arothuis) [inline graylog/Vagrantfile]: Ah, I see, thanks for the info. I guess we can remove ` VAGRANTFILE_API_VERSION = "2"`?

**Comment by arothuis** [comment]
👍

I had some troubles provisioning that seemed to be fixed by updating my `ubuntu/trusty64` vagrant box to the latest version (`20160708.1.0`). If we get problems in the future, we might want to explicitly pin down a version for the box.

File: docs/development.md [hunk 4]
```diff
@@ -9,8 +9,12 @@ and follow the installation instructions for each application.
 Useful Postman collections can be found on LastPass under `Shared-StepUp`.
 
 ## Graylog
-In order for Graylog2 to work, the correct source input should be configured. In the `System/Inputs` screen,
-add a GELF UDP with the configured settings, `port: 12201` and `bind_address: 0.0.0.0` by default.
+Graylog should work out of the box. You can log in using the username `admin` and the password `password`.
```
Comment (arothuis) [inline docs/development.md]: Username and password are listed in the `README.md`, not sure if it should be listed here as well.

Comment (DRvanR) [inline docs/development.md]: done

**Comment by DRvanR** [comment]
> If we were to have problems in the future, we might want to explicitly pin down a version for the box.
Agree.

File: graylog/Vagrantfile (hunk 3 above)
Comment (DRvanR) [inline graylog/Vagrantfile]: see https://www.pivotaltracker.com/story/show/126124891

**Pull request merged**